llm/llms.txt

```diff
@@ -0,0 +1,24 @@
+# Claudette
+
+> Claudette is a Python library that wraps Anthropic's Claude API to provide a higher-level interface for creating AI applications. It automates common patterns while maintaining full control, offering features like stateful chat, prefill support, image handling, and streamlined tool use.
+
+Things to remember when using Claudette:
+
+- You must set the `ANTHROPIC_API_KEY` environment variable with your Anthropic API key
+- Claudette is designed to work with Claude 3 models (Opus, Sonnet, Haiku) and supports multiple providers (Anthropic direct, AWS Bedrock, Google Vertex)
+- The library provides both synchronous and asynchronous interfaces
+- Use `Chat()` for maintaining conversation state and handling tool interactions
+- When using tools, the library automatically handles the request/response loop
+- Image support is built in but only available on compatible models (not Haiku)
+
+## Docs
+
+- [README](https://raw.githubusercontent.com/AnswerDotAI/claudette/refs/heads/main/README.md): Quick start guide and overview
+
+## API
```

Comment:
Question about api list and the intent of llm-txt. This is kind of tricky because it is hard to update both apilist and the ctx docs at the same time because llm-txt only points to the version which has been merged. Therefore the ctx and ctx-full content will always lag by maybe a PR.

It seems like there are a couple options:
1. If I could point the llms_ctx2txt to a local file sometimes then everything could be updated at the same time. Not sure if that is supported though.
2. llms-ctx and llms-ctx-full really should be generated by downstream clients and not explicitly included in the repos themselves.

What do you think?

Reply:
Now that I think about it, this isn't unique to apilist. Each one of these links to github point to a potentially outdated version with respect to what is being generated.